projects/1.19/assets/sweetmagic/sweetmagic/lang/zh_cn.json

```
@@ -0,0 +1,1097 @@
+	"block.sweetmagic.magia_rewrite": "魔力转写台",
+	"block.sweetmagic.aether_reverse": "以太反转台",
+	"block.sweetmagic.magia_drawer": "魔力抽屉",
+	"block.sweetmagic.aether_lanp": "以太灯笼",
+	"block.sweetmagic.aether_crystal_ore": "以太水晶矿石",
+	"block.sweetmagic.deep_aether_crystal_ore": "深层以太水晶矿石",
+	"block.sweetmagic.alternative_block": "亚钢块",
+	"block.sweetmagic.aethercrystal_block": "以太水晶块",
+	"block.sweetmagic.divinecrystal_block": "神圣水晶块",
+	"block.sweetmagic.purecrystal_block": "纯净水晶块",
+	"block.sweetmagic.aethercrystal_block_alpha": "以太水晶块",
+	"block.sweetmagic.sugarglass": "糖玻璃",
+	"block.sweetmagic.shading_sugarglass": "深色糖玻璃",
+	"block.sweetmagic.frosted_glass": "磨砂玻璃",
+	"block.sweetmagic.frosted_glass_line": "划线磨砂玻璃",
+	"block.sweetmagic.prismglass": "棱镜玻璃",
+	"block.sweetmagic.shading_prismglass": "深色棱镜玻璃",
+	"block.sweetmagic.green4panel_glass": "十字镶框玻璃（绿）",
+	"block.sweetmagic.lightbrown4panel_glass": "十字镶框玻璃（浅棕）",
+	"block.sweetmagic.brown4panel_glass": "十字镶框玻璃（棕）",
+	"block.sweetmagic.darkbrown4panel_glass": "十字镶框玻璃（深棕）",
+	"block.sweetmagic.ami_glass": "嵌丝玻璃",
+	"block.sweetmagic.gorgeous_glass": "玻璃窗（黑）",
+	"block.sweetmagic.gorgeous_glass_w": "玻璃窗（白）",
+	"block.sweetmagic.magicbarrier": "魔力屏障玻璃",
+	"block.sweetmagic.sugarglass_pane": "糖玻璃板",
+	"block.sweetmagic.shading_sugarglass_pane": "深色糖玻璃板",
+	"block.sweetmagic.frosted_glass_pane": "磨砂玻璃板",
+	"block.sweetmagic.frosted_glass_line_pane": "划线磨砂玻璃板",
+	"block.sweetmagic.prismglass_pane": "棱镜玻璃板",
+	"block.sweetmagic.shading_prismglass_pane": "深色棱镜玻璃板",
+	"block.sweetmagic.green4panel_glass_pane": "十字镶框玻璃板（绿）",
+	"block.sweetmagic.lightbrown4panel_glass_pane": "十字镶框玻璃板（浅棕）",
+	"block.sweetmagic.brown4panel_glass_pane": "十字镶框玻璃板（棕）",
+	"block.sweetmagic.darkbrown4panel_glass_pane": "十字镶框玻璃板（深棕）",
+	"block.sweetmagic.ami_glass_pane": "嵌丝玻璃板",
+	"block.sweetmagic.gorgeous_glass_pane": "玻璃窗板（黑）",
+	"block.sweetmagic.gorgeous_glass_w_pane": "玻璃窗板（白）",
+	"block.sweetmagic.moden_shelf_i": "现代柜（粉红）",
+	"block.sweetmagic.moden_shelf_l": "现代柜（黄绿）",
+	"block.sweetmagic.moden_shelf_o": "现代柜（橙）",
+	"block.sweetmagic.moden_shelf_p": "现代柜（紫）",
+	"block.sweetmagic.moden_shelf_r": "现代柜（红）",
+	"block.sweetmagic.moden_shelf_s": "现代柜（淡蓝）",
+	"block.sweetmagic.moden_shelf_y": "现代柜（黄）",
+	"block.sweetmagic.woodbox": "板条箱",
+	"block.sweetmagic.treasure_chest": "藏宝箱",
+	"block.sweetmagic.moden_rack": "现代货架（白）",
+	"block.sweetmagic.moden_rack_m": "现代货架（棕）",
+	"block.sweetmagic.moden_rack_t": "现代货架（浅棕）",
+	"block.sweetmagic.moden_rack_b": "现代货架（黑）",
+	"block.sweetmagic.wand_pedastal": "法杖基座",
+	"block.sweetmagic.corkboard": "软木展示框",
+	"block.sweetmagic.wallboard": "壁挂展示框",
+	"block.sweetmagic.wallboard_b": "壁挂展示框（黑）",
+	"block.sweetmagic.couter_table_sink_white_brick_r": "白线砖厨房水槽（红）",
+	"block.sweetmagic.iris_creation": "魔力坩埚",
+	"block.sweetmagic.mill": "磨粉机",
+	"block.sweetmagic.oven": "烤箱",
+	"block.sweetmagic.frypan_r": "平底锅（红）",
+	"block.sweetmagic.pot_w": "锅（白）",
+	"block.sweetmagic.bottle": "发酵瓶",
```

```suggestion
	"block.sweetmagic.bottle": "发酵罐",
```
[术语在线](https://www.termonline.cn/search?searchText=%E5%8F%91%E9%85%B5%E7%BD%90)